TheMovieDb/Helpers/Constants.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  Constants.swift
+//  TheMovieDb
+//
+//  Created by Karla Rubiano on 3/11/21.
+//
+
+import Foundation
+
+struct Constants {
+    static let apiKey = "f6cd5c1a9e6c6b965fdcab0fa6ddd38a"
+    static let language = "en-US"
+    static let reviewsTitleBarButton = "Reviews"
+    static let cellIdentifier = "cell"
+    static let reviewDetailStoryboardID = "reviewsDetailID"
+    static let viewControllerStoryboardID = "Detail"
+    static let searchTableViewControllerID = "Search"
+    static let movieInfoViewControllerID = "MovieInfo"
+    static let reviewsViewControllerID = "Reviews"
+    static let titleInitialTableView = "Show Movies"
+    static let searchBarPlaceholder = "Search for movies or person"
+    static let personDetailViewControllerID = "PersonDetail"
+    static let errorAlertTitle = "Error"
+    static let errorAlertMessage = "Service Failure"
+    static let errorAlertButton = "Dismiss"
+    static let imageBaseUrl = "https://image.tmdb.org/t/p/w500"
+}
```

Comment:
As you can see. The constants file keeps growing and one of the approaches that you can do is divide and conquer.  

- In the case of the apiKey and language those are parameters for our request 
- In the case of Titles we will discuss about the localizable files the next session by now you can left them here.
- For the view controller now you can easily change an identifier here maybe you can be more explicit in the strings.
- In the case of the errors, you can make an enum to conform the protocol Error and make that error to have a string.
- For the images The MovieDB supports a lot of resolutions maybe in the future we can add an image resolution support for X type of device.

Reply:
Separated the constants and fixed the error titles and messages.